### DISCUSSION TITLE
Will NetBox shift from Redis to Valkey?

### DISCUSSION THREAD

**Discussion by ghost** [netbox-community/netbox#17334] [General]
Reference: https://news.slashdot.org/story/24/09/01/072220/open-source-redis-fork-valkey-has-momentum-improvements-and-speed-says-dirk-hohndel

I know next to nothing about upstream Redis. But Valkey appears to have a lot of support and highly requested multi-threaded improvements coming.
Are there any plans to consider the move from Redis to Valkey?

**Comment by alehaa** [comment]
In https://github.com/netbox-community/netbox/discussions/15500#discussioncomment-8882468 it seems like there‘s no need to change.

**Reply by candlerb** [reply]
[Valkey now replaces Redis in Ubuntu 24.10](https://lists.ubuntu.com/archives/ubuntu-announce/2024-October/000307.html), so users may be pushed towards Valkey not by licensing issues but by package availability (at least by the time 26.04 LTS comes along)
